clients/client-python/build.gradle.kts

```diff
@@ -21,11 +22,29 @@ fun deleteCacheDir(targetDir: String) {
   }
 }
 
+fun waitForPort(port: Int, host: String = "localhost", timeout: Long = 60000) {
```

Comment:
I'm not sure only waiting for port is enough, should we check the URI like version?

Reply:
It would be very complicated if we enabled security to check the version.

Reply:
how about /metrics? it's not supposed to add security check

Reply:
Let me take a look.

Reply:
@FANNG1 
<img width="1143" alt="image" src="https://github.com/datastrato/gravitino/assets/15794564/4be7ee64-f061-4cc2-a519-f39dfef3a2ca">
It seems that `/metrics` also be protected by `authenticaion`.

Reply:
@qqqttt123 , seems not expected ?

Reply:
@qqqttt123 Could you help me verify this point?

Reply:
@FANNG1
Please review agian